packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -526,6 +589,21 @@ class VideoViewport extends Viewport implements IVideoViewport {
     }
   }
 
+  public getCurrentImageId() {
+    const current = this.imageId.replace(
+      '/frames/1',
+      this.isPlaying
+        ? `/frames/1-${this.numberOfFrames}`
+        : `/frames/${this.getFrame()}`
+    );
+    return current;
+  }
```

Comment:
Does the imageId always have /frames/1? what about a direct link to an mp4 that we initially started with